backend/experiment/rules/musical_preferences.py

```diff
@@ -230,38 +236,9 @@ def calculate_score(self, result, data):
         else:
             return super().calculate_score(result, data)
 
-    def social_media_info(self, block, top_participant, known_songs, n_songs, top_all):
-        """⚠️ Deprecated. The social media info will eventually be present on the Experiment level, not the Block level."""
-        current_url = "{}/{}".format(settings.RELOAD_PARTICIPANT_TARGET, block.slug)
-
-        experiment = block.phase.experiment
-        social_media_config = experiment.social_media_config
-        tags = social_media_config.tags if social_media_config.tags else []
-        url = social_media_config.url or current_url
-
-        def format_songs(songs):
-            return ", ".join([song["name"] for song in songs])
-
-        return {
-            "apps": ["weibo", "share"],
-            "message": _(
-                "I explored musical preferences on %(url)s! My top 3 favorite songs: %(top_participant)s. I know %(known_songs)i out of %(n_songs)i songs. All players' top 3 songs: %(top_all)s"
-            )
-            % {
-                "url": current_url,
-                "top_participant": format_songs(top_participant),
-                "known_songs": known_songs,
-                "n_songs": n_songs,
-                "top_all": format_songs(top_all),
-            },
-            "url": url,
-            "hashtags": [*tags, "amsterdammusiclab", "citizenscience"],
-        }
-
     def get_final_view(self, session, top_participant, known_songs, n_songs, top_all):
         # finalize block
-        social_info = self.social_media_info(session.block, top_participant, known_songs, n_songs, top_all)
-        social_info["apps"] = ["weibo", "share"]
+        social_info = self.social_media_info(session)
```

Comment:
The channels of these rules files are different from base (i.e., "weibo", "share"), those should still be adjusted.

Reply:
Ah, nevermind. Realising now it's set through the database.